README.md

```diff
@@ -17,39 +17,50 @@ https://cloud.google.com/functions/docs/first-java#gradle_1
 
-This workflow made certain assumptions about the cloud function runtime service account that users should be familiar with.
-It also made certain custom parameters available to `java11` runtime as `envvar`'s that overcomes some limitations in the current Google Cloud Function `java11` binding.
-Ultimately all custom values will come from `deltalayer` as `tfvar`'s so that the cloud function code can be built once and deployed as various cloud functions across different environments.
-For more information on `deltalayer`, please refer to
-* [terraform-ap-modules](https://github.com/broadinstitute/terraform-ap-modules/tree/master/deltalayer) and
-* [terraform-ap-deployments](https://github.com/broadinstitute/terraform-ap-deployments/tree/master/deltalayer).
+Please refer to `deploy-testrunner-dev-streamer.yml` for a working example.
 
-| Name | Description | Type | Default | Required |
-|------|-------------|------|---------|:--------:|
-| `GCLOUD_PROJECT` | Specify the Google Project ID that the Cloud Function deploys to. | `string` | n/a | yes |
-| `service-account` | The runtime Service Account email assumed by Cloud Function | `string` | `PROJECT_ID@appspot.gserviceaccount.com` | no |
-| `GOOGLE_APPLICATION_CREDENTIALS` | The Application Default Credentials that possess `cloudfunction.admin` role | `string` | n/a | yes |
-| `NAME` | The ID of the function or fully qualified identifier for the function | `string` | n/a | yes |
-| `trigger-resource` | The name of the google storage bucket (the string after gs://) that triggers cloud function for consumption. | `string` | n/a | yes |
-| `trigger-event` | Specifies which action should trigger the function. For a list of acceptable values, call `gcloud functions event-types list`. | `string` | n/a | yes |
-| `entry-point` | The fully qualified Java class name of the function. | `string` | n/a | yes |
-| `set-env-vars` | List of comma-separated key-value pairs to set as environment variables. All existing environment variables will be removed first. | `string` | n/a | no |
+The `deploy-cloud-function` action requires a deployer service account (identified as `deployer-sa-key`, `deployer-sa-email`) for cloud function deployment and an identity (`service-account`) that the cloud function assumes at runtime.
+These service accounts are provisioned by Terraform and their secrets are stored in the vault.
+
+For more information, please refer to
+* [deltalayer module](https://github.com/broadinstitute/terraform-ap-modules/tree/master/deltalayer)
+* [deltalayer deployments](https://github.com/broadinstitute/terraform-ap-deployments/tree/master/deltalayer)
+* [testrunner module](https://github.com/broadinstitute/terraform-ap-modules/tree/master/testrunner)
+* [testrunner deployments](https://github.com/broadinstitute/terraform-ap-deployments/tree/master/testrunner)
+
+## Moving Vault secrets to Github via Atlantis
```

Comment:
Does this describe moving a secret (meaning the SoT is now a GH secret), or is this a sync process that copies a secret from Vault to GH (meaning the SoT is Vault)?

Reply:
This refers to the sync process, the README has been updated to state this fact more clearly.